src/web/src/app/app.component.ts

```diff
@@ -133,20 +137,24 @@ export class AppComponent implements OnInit, OnDestroy {
 	}
 
 	ngOnInit() {
-		this.auth.isAuthorized().subscribe(ok => {
+		this._subAuth = this.auth.isAuthorized().subscribe(ok => {
 			this.authorized = ok;
 			this.changeDetectorRef.detectChanges();
 		});
 
-		this.space.isInitialized().subscribe(ok => {
+		this._subSpace = this.space.isInitialized().subscribe(ok => {
 			this.initialized = ok;
 			this.changeDetectorRef.detectChanges();
 		});
+
+		this.activeBadge = this.isSpacesBadge();
```

Comment:
invitationSize будет приходить с бэка.
В userNotificationSize суммируем все уведомления для пользователя, пока это только invitationSize, но есть идея добавить ещё =)

```suggestion
		this._subSpace = this.space.isInitialized().subscribe(res => {
			if (!res)
				return;
			this.initialized = true;
			const invitationSize = 1; // res.invitationSize
			this.spaceInvitationSize = invitationSize;
			this.userNotificationSize += invitationSize;
			this.changeDetectorRef.detectChanges();
		});
```

Reply:
Дубль. Готово.